.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,33 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: 'The name of the Tarantool build artifact'
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: 'Clone the expirationd module'
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/expirationd
+
+      - name: 'Download the Tarantool build artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: 'Install Tarantool'
+        # TODO(ylobankov): Install package dependencies. Now we're lucky: all
```

Comment:
If we'll add a runtime dependency into tarantool and it appears that it is not installed by default on GitHub virtual runners, we'll hit by the problem here (and in other modules): `dpkg -i file` does not install dependencies. So I would consider it as problematic code that should be fixed sooner or later.

It is easy for CentOS / Fedora (just use `yum install -y file` instead of `rpm -i file`). We should find something like this for Debian / Ubuntu (maybe use apt? whether it is available on Debian or it is pure Ubuntu tool?).

The question was raised first time [here](https://github.com/tarantool/smtp/blob/ed6434629cd0ab4514de13e5e1780a03903f403b/.github/workflows/packaging.yml#L146-L166).

Anyway, it is up to @ligurio how to better present this information here. I just share my opinion.